react-example/src/index.tsx

```diff
@@ -38,35 +41,19 @@ const HomeLink = styled(Link)`
 `;
 
 ((): void => {
-  const initializeParams: WithJWTParams = {
+  // Here we are testing it using NON-JWT based project.
+  //JWT based project works but we assumed that generateJWT function will take-in userId as param to generate JWT
+  const initializeParams: WithoutJWTParams = {
     authToken: process.env.API_KEY || '',
     configOptions: {
       isEuIterableService: false,
-      dangerouslyAllowJsPopups: true
-    },
-    generateJWT: ({ email }) => {
-      return axios
-        .post(
-          process.env.JWT_GENERATOR || 'http://localhost:5000/generate',
-          {
-            exp_minutes: 2,
-            email,
-            jwt_secret: process.env.JWT_SECRET
-          },
-          {
-            headers: {
-              'Content-Type': 'application/json'
-            }
-          }
-        )
-        .then((response) => {
-          return response.data?.token;
-        });
```

Comment:
We should not just remove this. We need to think of a way to test both `WithJWT` and `WithoutJWT` instead.

Reply:
Done